components/core/CarouselSidebarData.js

```diff
@@ -0,0 +1,758 @@
+import * as React from "react";
```

Comment:
CarouselSidebarData.js and CarouselSidebarSlate.js have been removed and replaced with CarouselSidebar.js

If there's any changes you made to this, please add them to CarouselSidebar.js instead and delete this